packages/next-urql/examples/5-with-suspense-exchange/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "5-with-suspense-exchange",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "next dev",
+    "prestart": "cd ../../ && yarn build && cd examples/3-with-custom-exchange"
```

I think we need to update this to:

```suggestion
    "prestart": "cd ../../ && yarn build && cd examples/5-with-suspense-exchange"
```

Let's verify that these changes still work with the correct example getting spun up.

---

lol, when i was initially working on this, i added suspense to example 3. that must have been why running the start command from the copy-pasta'd example 5 worked as expected 😅